server/services/s3Upload.js

```diff
@@ -0,0 +1,41 @@
+const AWS = require('aws-sdk');
+AWS.config.update({
+  region: '',
+  credentials: {
+    accessKeyId: '',
+    secretAccessKey: '',
+  },
+});
+
+const s3 = new AWS.S3({apiVersion: '2006-03-01'});
+
+//TODO:
```

Comment:
you have use the same structure. uuid will give it a unique string and then you can add the userId to it.